docs/basics/acct-team-mgmt/test-object.md

```diff
@@ -7,7 +7,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-<p><span className="sauceGold">DEPRECATED</span></p>
+<p><span className="sauceRed">REMOVED</span></p>
```

Comment:
Hi @sweeneyskirt-sl ,
Can you let me know the reason we mention TO as REMOVED and not completely erase it from the docs?

Reply:
Hey @linafarid -- At this time, we are still working on our internal docs redirect solution, so we have a freeze on physical removal of any page in the docs that has been published to production, and is, therefore, indexed. If anyone has linked to this page externally (including our own marketing site or the Sauce Labs app), we would rather they hit this page and have an explanation that hit a 404. I have removed this page from our navigation, as well as all links to it from our docs, so the only way someone would reach it is by clicking an external link or entering the URL manually.